.github/workflows/ci.yaml

```diff
@@ -97,7 +98,6 @@ jobs:
         flavor: |
           latest=false
         tags: |
-            type=raw,value=latest,enable=${{ endsWith(github.ref, github.event.repository.default_branch) }}
```

Comment:
Is this image tag removal intended or did it leak in from a different change?

Reply:
That definitely leaked in. I still want it though.